support-frontend/assets/helpers/redux/checkout/product/selectors/isSupporterPlus.test.ts

```diff
@@ -0,0 +1,33 @@
+import { _ } from './isSupporterPlus';
+
+describe('isInHideBenefitsTest', () => {
```

Comment:
Jest looks extremely useful testing standalone calls, will use this in future for standalone function testing. Have 'run' the test manually/successfully from visual studio, this brings up a jest console. Are these jest lines added to the build automatically as well?

Reply:
@paul-daniel-dempsey yes unit tests are a nice way of validating your code does what you expect it to, and yes we have a Github action that runs these tests on our Pull Requests too!